rootfs/api/settings/production.py

```diff
@@ -260,6 +260,10 @@
 SECRET_KEY = os.environ.get('DEIS_SECRET_KEY', random_secret)
 BUILDER_KEY = os.environ.get('DEIS_BUILDER_KEY', random_secret)
 
+# experimental native ingress
+EXPERIMENTAL_NATIVE_INGRESS = os.environ.get('EXPERIMENTAL_NATIVE_INGRESS', 0)
```

Comment:
we will need to convert this to a bool in order for this to work, like so

```
EXPERIMENTAL_NATIVE_INGRESS = bool(os.environ.get('EXPERIMENTAL_NATIVE_INGRESS', 0))
```

os.environ.get('EXPERIMENTAL_NATIVE_INGRESS', 0) will be of type string or int otherwise.